src/libEth/Eth.cpp

```diff
@@ -128,7 +128,7 @@ EthFields parseRawTxFields(std::string const& message) {
         ret.amount = uint128_t(*it);
         break;
       case 5:
-        ret.data = byteIt;
+        ret.code = byteIt;
```

Comment:
I will mention on slack - Eth TXs only have a code field (not code and data)

Reply:
how come? They have code and calldata?